Assets/Scripts/Models/Furniture.cs

```diff
@@ -209,15 +238,29 @@ private set
         }
     }
 
+    /// <summary>
+    /// Gets or sets a value indicating whether the furniture is selected by the player or not.
+    /// </summary>
+    /// <value>Whether the furniture is selected or not.</value>
     public bool IsSelected { get; set; }
 
-    // This represents the BASE tile of the object -- but in practice, large objects may actually occupy
-    // multile tiles.
+    /// <summary>
+    /// Gets the BASE tile of the furniture. (Large objects can span over multiple tiles).
+    /// </summary>
+    /// <value>The BASE tile of the furniture.</value>
     public Tile Tile { get; private set; }
```

Comment:
Yes, property names is really important, I will add that this needs to be renamed in the documentation, but we have not agreed completely on how we will do that. I will add a separate PR after this one that will deal with the renaming of everything in Furniture .cs